interpreter/cling/lib/Interpreter/DefinitionShadower.cpp

```diff
@@ -83,8 +92,10 @@ namespace cling {
     for (auto Prev : Previous) {
       if (Prev == D)
         continue;
+#ifdef CLING_ONLY_SHADOW_USER_DECLS
       if (!isClingShadowNamespace(Prev->getDeclContext()))
         continue;
+#endif
```

Comment:
I think we can just get rid of this?

Reply:
I just included it just in case one wants to compile Cling using the old behavior.  Otherwise, we can get rid of it.